db/migrate/20230502183211_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] Great job @dennis-every, However, I would suggest you change the biography type to `text` instead of a string.
- Note: In PostgreSQL, both text and string data types can be used to store strings of characters. The main difference between the two is their maximum storage size and the way they handle trailing whitespace.
- `text` is an unlimited-length character type, which means it can store strings of any length. `string` is a variable-length character type, which means that it can store strings up to a maximum length specified when the column is created.